docs/source/conf.py

```diff
@@ -0,0 +1,54 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = "Lattice Surgery Compiler"
+copyright = "2021, George Watkins and Alex Nguyen"
+author = "George Watkins and Alex Nguyen"
```

Comment:
At some point we may need to sort out the copyright/authorship situation. We might want change this `George Watkins, Alex Nguyen and Contributors`,then have an [`AUTHORS` file](https://opensource.google/docs/releasing/authors/) where we acknowledge advisors and other contributors like Varun and Keelan

Reply:
I will look into it